src/main/groovy/org/jetbrains/intellij/dependency/IdeaDependencyManager.groovy

```diff
@@ -295,13 +295,17 @@ class IdeaDependencyManager {
         def configurations = project.configurations
         def closure = {
             if ("org.jetbrains.kotlin" == it.group) {
-                return "kotlin-runtime" == it.name || it.name.startsWith('kotlin-stdlib') || "kotlin-reflect" == it.name
+                return isKotlinRuntime(it.name)
             }
             return false
         }
         return configurations.getByName(JavaPlugin.RUNTIME_CONFIGURATION_NAME).getAllDependencies().find(closure) ||
                 configurations.getByName(JavaPlugin.COMPILE_CONFIGURATION_NAME).getAllDependencies().find(closure)
     }
+
+    static boolean isKotlinRuntime(name) {
+        return "kotlin-runtime" == name || name.startsWith('kotlin-stdlib') || "kotlin-reflect" == name
```

Comment:
the first usage (in `IdeaDependency`) matches file names, `kotlin-rutime == name` will be always false for `kotlin-runtime.jar` file. I think we can do something like `file.name - '.jar'` in the first usage